src/Microsoft.Identity.Web/DownstreamWebApiSupport/MicrosoftIdentityAppAuthenticationMessageHandler.cs

```diff
@@ -29,6 +30,22 @@ public class MicrosoftIdentityAppAuthenticationMessageHandler : MicrosoftIdentit
         {
         }
 
+        /// <inheritdoc/>
+        protected override async Task<AuthenticationResult> GetTokenAsync(MicrosoftIdentityAuthenticationMessageHandlerOptions options)
+        {
+            if (options == null)
+            {
+                throw new ArgumentNullException(nameof(options));
+            }
+
+            return await TokenAcquisition.GetAuthenticationResultForAppAsync(
+                options.Scopes!,
+                options.AuthenticationScheme,
+                options.Tenant,
+                options.TokenAcquisitionOptions)

```

Comment:
The user flow is no longer included.

Reply:
Sorry, can you expand on the issue? In the app case, I don't see the user flow in the old code either. I do include the user flow in the "ForUser" case.

If I'm missing something, please let me know. Thanks!